content/guides/getstarted/how-to-create-a-token.md

```diff
@@ -0,0 +1,329 @@
+| Web3 Storage         | [web3.storage](https://web3.storage)                     | Requires signing up for a free plan; first 5Gb are free; easy to use.                               |
+| Akord                | [Akord](https://akord.com/)                              | Uploads to arweave; free without signup for 100Mb; uploads can take a while.                        |
+| GitHub               | [GitHub](https://github.com)                             | Create a new repo, upload files, use the RAW URL format: https://raw.githubusercontent.com/xxxxxxx. |
+| ShadowDrive          | [ShadowDrive](https://www.shdwdrive.com/)                | This is a solana native storage solution. Did not work at the time of writing.                      |
+| Metaplex Metaboss    | [Metaplex Metaboss](https://metaplex.com/)               | [Meta boss docs](https://metaboss.rs/)-                                                             |
+| Google Cloud Storage | [Google Cloud Storage](https://cloud.google.com/storage) | -                                                                                                   |
+| Amazon               | [Amazon](https://aws.amazon.com/)                        | -                                                                                                   |
+
+(If you are a provider of decentralized storage solutions and want to be listed
+here please just open a PR using the edit page button)
+
+For this example we will use Web3 Storage. It requires to sign up for a free
+plan though but the first 5Gb are free and its very easy to use.
+
+- [web3.storage](web3.storage)
+
+Once we have generated and uploaded the icon to a decentralized storage solution
+we end up with a link to that icon. We will use that link in the meta data file.
+
+You will end up with some link similar to this one which should directly open
+your icon: https://arweave.net/itK2SKyCDAdBl-t9sHDQzeP4Roh3UgMaqnKImRXSrvo
+
+Create a new file called `metadata.json` in the `nice-token` folder and add the
+following:
+
+```json
+{
+  "name": "Nice Token",
+  "symbol": "NICE",
+  "image": "https://arweave.net/itK2SKyCDAdBl-t9sHDQzeP4Roh3UgMaqnKImRXSrvo"
+}
+```
+
+Then also upload this file to the storage provider of your choice. You will end
+up with a new link that now represents the meta data of your token. Make sure
+you copy the link directly to the file and not to the folder or a preview page.
+
+You should end up with a link similar to this one:
+
+https://bafybeidfm65jzvz4zeesxp6ybinkitvpd27klk6yspstrtw5fuy5w27lkq.ipfs.w3s.link/metadata.json
+
+Now we are ready to add this meta data to our token.
+
+## Add meta data
+
+Now we will initialize the meta data for our token with the meta data we just
+created and uploaded.
+
+```bash
+spl-token initialize-metadata niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun \
```

Comment:
I see you using these multi line shell commands now :)

Reply:
Yep great tip, thanks